sites/docs/docs/tutorial-plugin-coordination-type.mdx

```diff
@@ -4,13 +4,13 @@ title: 'Tutorial: Develop a plugin coordination type'
 sidebar_label: Plugin Coordination Type
 ---
 
-See the [plugin coordination type development](/docs/dev-plugins/#plugin-coordination-types) documentation for more details.
+See the [plugin coordination type development](https://github.com/vitessce/vitessce/blob/main/dev-docs/plugin-view-types.md) documentation for more details.
 
 ### Minimal example
 
 ```js
 import React from 'react';
-import Slider from '@material-ui/core/Slider';
+import Slider from '@material-ui/core/es/Slider/index.js';
```

Comment:
Why does this need to happen now?

Reply:
I guess this is my concern - how much are we changing here, not just in our own codebase but requirements that other codebases do certain things?